src/envo/envo-edit.owl

```diff
@@ -2942,6 +2942,7 @@ Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_01001621>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_01001622>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_01001623>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_01001624>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_01001625>))
```

Comment:
This, and the accompanying stanzas in the Class section, was what was causing the conflict.